mitosheet/mitosheet/step_performers/pivot.py

```diff
@@ -110,6 +112,7 @@ def execute(cls, prev_state: State, params: Dict[str, Any]) -> Tuple[State, Opti
             # actually used by the backend (which is kinda confusing and should be renamed)
             'destination_sheet_index': final_destination_sheet_index,
             'new_df_name': new_df_name,
+            'extra_code': params.get('extra_code')
```

Comment:
Should not put this in the execution data - I think this should just be a straight up param. We duplicate some work here